src/Spectre.sln

```diff
@@ -103,112 +103,164 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Spectre.Dependencies", "Spe
 EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
```

Comment:
There should be no platform configuration for `Any CPU`, only for `x64`. This relates to everything that is below in this file.